tools/release/java/build.xml

```diff
@@ -62,29 +58,56 @@
             <arg value = "${newdir}"/>
 
             <arg value = "--cppxslt"/>
-            <arg value = "${basedir}/src/com/ibm/icu/dev/tools/docs/dumpAllCppFunc.xslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/dumpAllCppFunc.xslt"/>
 
             <arg value = "--cxslt"/>
-            <arg value = "${basedir}/src/com/ibm/icu/dev/tools/docs/dumpAllCFunc.xslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/dumpAllCFunc.xslt"/>
 
             <arg value = "--reportxslt"/>
-            <arg value = "${basedir}/src/com/ibm/icu/dev/tools/docs/genReport.xslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/genReport.xslt"/>
 
             <arg value = "--resultfile"/>
             <arg value = "${basedir}/APIChangeReport.html"/>
 
-            <classpath>
-                <pathelement location="${build.dir}"/>
-                <pathelement path="${java.class.path}/"/>
-                <pathelement location="${lib.dir}/xalan.jar"/>
-		<fileset dir="${lib.dir}"/>
-                <pathelement location="${lib.dir}/xercesImpl.jar"/>
-                <pathelement path="."/>
-            </classpath>
+        </java>
+    </target>
+
+    <target name="apireport_md" depends="tools">
+        <java jar="${apireport.jar}" fork="yes" failonerror="true">
+
+<!--
+            <arg value = "-DELETEME-newver"/>
+            <arg value = "4.6"/>
+-->
+
+            <arg value = "--olddir"/>
+            <arg value = "${olddir}"/>
+
+<!--
+	!! Only need these if you must override. Remove the DELETEME
+            <arg value = "-DELETEME-newver"/>
+            <arg value = "${newver}"/>
+-->
+
+            <arg value = "--newdir"/>
+            <arg value = "${newdir}"/>
+
+            <arg value = "--cppxslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/dumpAllCppFunc.xslt"/>
+
+            <arg value = "--cxslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/dumpAllCFunc.xslt"/>
+
+            <arg value = "--reportxslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/genReport_md.xslt"/>
+
+            <arg value = "--resultfile"/>
+            <arg value = "${basedir}/APIChangeReport.md"/>
+
         </java>
     </target>
     <target name="apireport_xml" depends="tools">
-        <java classname="com.ibm.icu.dev.tools.docs.StableAPI" fork="yes" failonerror="true">
+        <java jar="${apireport.jar}" fork="yes" failonerror="true">
 
 <!--
```

Comment:
I'm definitely not a fan of these uncommented "magic" comments. It feels like they'll just rot over time.
Could we have a flag that's always set but takes a value (e.g. --newver=${want.new.version}") and then the property is overrideable normally on the command line instead of having to edit the build file.

Reply:
> . It feels like they'll just rot over time.

they already have rotted. It's probably better just to remove them entirely from ant and document that they can be set manually

Reply:
dropping them